core/modules/node/config/views.view.node_admin_content.json

```diff
@@ -3,7 +3,7 @@
     "name": "node_admin_content",
     "description": "Administrative listing for managing content.",
     "module": "node",
-    "storage": 4,
+    "storage": 2,
```

Comment:
I think it needs to stay as 4. 2 means it's no longer default I believe.

Reply:
Good catch.